src/components/About/ProfileCard.module.css

```diff
@@ -0,0 +1,87 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  padding-top: 2rem;
+  padding-left: 1.5rem;
+  padding-right: 1.5rem;
+  height: 13.813rem;
+  width: 18.875rem;
+  background: var(--dark-theme-background);
+}
+
+.avatar {
+  width: 96px;
+  height: 96px;
+  border-radius: 50%;
+  border-color: #C4FFFE;
+  border-width: 2px;
+  border-style: solid;
+  margin-bottom: 1rem;
+}
+
+.name {
+  color: var(--rapture-blue);
+  width: 100%;
+  text-align: center;
+}
+
+.title {
+  color: var(--primary-white);
+  font-style: italic;
+  font-weight: 500;
```

Comment:
Currently, the 500 weight of the font is not being imported, so the display doesn't actually reflect this.

The Figma design does use medium italic here (500 weight) despite it not being mentioned in the typography specs. To update the font import, change the import URL in line 1 of `App.css` to `https://fonts.googleapis.com/css2?family=Livvic:ital,wght@0,500;0,600;0,700;1,500;1,600;1,700&display=swap` (adding the regular and italic for weight 500)